### ISSUE TITLE
put j_bias and alpha in LIFNeuron

### ISSUE THREAD

**Issue by jaberg** [ctn-archive/nengo_theano#38]
j_bias and alpha seem to me like natural attributes of the neuron, which should be used in LIFNeuron's update function. LIFNeuron already implements the logic for allocating these variables. What's the rationale for making them part of the enclosing ensemble, and accessing them from EnsembleOrigin too?

**Comment by studywolf** [comment]
So I've tried putting them in both places, but I think it does end up making more sense not having them in the neuron, but in the ensemble, because they're NEF dealies. In the NEF, the activity to the neuron is calculated as 

G[alpha \* enc \* X + j_bias], 

where then G is the neuron dynamics. So while they're calculated once and then assigned to specific neurons, every neuron model will still have these parameters, and if we get terry's generic calculation of the gain and bias then we can find them independent of the neuron model. 

tl;dr they're NEF things! independent of the neuron model. should be in ensemble, not neuron models

**Comment by jaberg** [comment]
I'm convinced, thanks for spelling it out.
